components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -103,63 +103,57 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.store.select(routeParams).pipe(
+      takeUntil(this.isDestroyed),
       pluck('id'),
       filter(identity),
-      takeUntil(this.isDestroyed))
```

Comment:
Refactored this code to not have a subscribe inside of a subscribe.

Reply:
whoa